Tests/MapboxNavigationTests/CarPlayManagerTests.swift

```diff
@@ -1,441 +1,276 @@
-                    CLLocationCoordinate2D(latitude: 34.054081, longitude: -118.243412),
-                ])
-                fakeRouteChoice.userInfo = (Fixture.routeResponse(from: "route-with-banner-instructions", options: options), 0, options)
-                let fakeTrip = CPTrip(origin: MKMapItem(), destination: MKMapItem(), routeChoices: [fakeRouteChoice])
-
-                //simulate starting a fake trip
-                manager!.mapTemplate(fakeTemplate, startedTrip: fakeTrip, using: fakeRouteChoice)
-                _ = manager!.carPlayNavigationViewController!.view
-                let service = delegate!.currentService! as! MapboxNavigationService
-                service.start()
-            }
-
-            context("When configured to simulate") {
-                beforeEach {
-                    manager!.simulatesLocations = true
-                    manager!.simulatedSpeedMultiplier = 5.0
-                }
-
-                it("starts navigation with a navigation service with simulation enabled") {
-                    action()
-
-                    expect(delegate!.navigationInitiated).to(beTrue())
-                    let service: MapboxNavigationService = delegate!.currentService! as! MapboxNavigationService
-
-                    expect(service.simulationMode).to(equal(.always))
-                    expect(service.simulationSpeedMultiplier).to(equal(5.0))
-                }
-            }
-
-            context("When configured not to simulate") {
-                beforeEach {
-                    manager!.simulatesLocations = false
-                }
-
-                it("starts navigation with a navigation service with simulation set to inTunnels by default") {
-                    action()
-
-                    expect(delegate!.navigationInitiated).to(beTrue())
-                    let service: MapboxNavigationService = delegate!.currentService! as! MapboxNavigationService
-
-                    expect(service.simulationMode).to(equal(.inTunnels))
-                }
-            }
-        }
-    }
-
-    private class CustomTripPreviewDelegate: CarPlayManagerDelegate {
-        var customTripPreviewTextConfiguration: CPTripPreviewTextConfiguration?
-        var customTrip: CPTrip?
-
-        func carPlayManager(_ carPlayManager: CarPlayManager, willPreview trip: CPTrip) -> CPTrip {
-            return customTrip ?? trip
-        }
-
-        func carPlayManager(_ carPlayManager: CarPlayManager, willPreview trip: CPTrip, with previewTextConfiguration: CPTripPreviewTextConfiguration) -> CPTripPreviewTextConfiguration {
-            return customTripPreviewTextConfiguration ?? previewTextConfiguration
-        }
-
-        func carPlayManager(_ carPlayManager: CarPlayManager, didBeginNavigationWith service: NavigationService) {
-            //no-op
-        }
-
-        func carPlayManagerDidEndNavigation(_ carPlayManager: CarPlayManager) {
-            //no-op
-        }
```

Comment:
No-op delegate methods are not needed, removed them in `CarPlayManagerSpec.swift`.